web/views/student.py

```diff
@@ -808,6 +823,40 @@ def dump_comment(comment):
         except KeyError as e:
             logging.exception(e)
 
+    # add comments from llm summary
+    if is_teacher(request.user):  # Currently only teachers can view LLM summary comments
+        llm_summary = getattr(resultset["summary"], "summary", "")
```

Comment:
Could you please refactor the `get` function in a separate PR first so that it returns a properly typed dataclass? This is becoming too brittle..